app/models/test_track/assignment.rb

```diff
@@ -16,7 +16,11 @@ def variant
   end
 
   def unsynced?
```

Comment:
i think we should rename this to `should_sync?`. since some `Assignment`s are no longer meant to be synced, it seems to me that this name is misleading as is. i.e. feature gates are intentionally "unsynced" so changing the naming here will provide a clearer indication of that.